src/generator/operations.ts

```diff
@@ -256,7 +256,8 @@ function generateOperation(op: Operation, imports: ImportManager): string {
   }
   let opName = op.language.go!.name;
   if (isLROOperation(op)) {
-    opName = opName[0].toLocaleLowerCase() + opName.substr(1);
+    // uncapitalizing runs the risk of reserved name collision, e.g. Import -> import
+    opName = getEscapedReservedName(opName.uncapitalize(), 'Operation');
```

Comment:
Question, is this something we should do in the namer?

Reply:
We could, it gets a little complex.  In the example above, the namer properly calls this method `Import` so there's no collision.  The only reason we have the collision is for LROs, where we replace `Import` with `BeginImport` and unexport the `Import` method.  Hmm, maybe we could add this info to `protocolNaming`.  I will look into that.

Reply:
fixed